### PR TITLE
App 5330 eiir synchronization of iir data to azure ai search

### PR DESCRIPTION
**Pull request by cmcintyre-inss**

### CODE REVIEW

File: INSS.EIIR.DataSync.Infrastructure/INSS.EIIR.DataSync.Infrastructure.csproj [hunk 1]
```diff
@@ -7,16 +7,19 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\INSS.EIIR.AzureSearch.IndexMapper\INSS.EIIR.AzureSearch.IndexMapper.csproj" />
     <ProjectReference Include="..\INSS.EIIR.DataSync.Application\INSS.EIIR.DataSync.Application.csproj" />
     <ProjectReference Include="..\INSS.EIIR.Models\INSS.EIIR.Models.csproj" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="6.0.0" />
```
Comment (cmcintyre-inss) [inline INSS.EIIR.DataSync.Infrastructure/INSS.EIIR.DataSync.Infrastructure.csproj:16]: double checked - is the latest version

File: INSS.EIIR.DataSync.Infrastructure/INSS.EIIR.DataSync.Infrastructure.csproj [hunk 2]
```diff
@@ -7,16 +7,19 @@
...
     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.8">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.8" />
+    <PackageReference Include="System.Linq.Async" Version="6.0.1" />
```
Comment (cmcintyre-inss) [inline INSS.EIIR.DataSync.Infrastructure/INSS.EIIR.DataSync.Infrastructure.csproj:22]: double checked - is the latest version

File: deploy-jobs-template.yml [hunk 3]
```diff
@@ -171,7 +171,7 @@ jobs:
                       },                      
                       {
                         "name": "UseFakedDataSources",
-                        "value": "$(UseFakedDataSources)",
+                        "value": "$(UseFakedDataSourcesInFunction)",
```
Comment (john5macd) [inline deploy-jobs-template.yml:174]: Suggest this is changed to UseFakedDataSourcesFuncEiir

File: deploy-jobs-template.yml [hunk 4]
```diff
@@ -348,7 +348,7 @@ jobs:
                       },                      
                       {
                         "name": "UseFakedDataSources",
-                        "value": "$(UseFakedDataSources)",
+                        "value": "$(UseFakedDataSourcesInExtract)",
```
Comment (john5macd) [inline deploy-jobs-template.yml:351]: Suggest this is changed to UseFakedDataSourcesFuncEiirDailyExtract to these variable names a function app based - and so where fakes are used they are either on or off

File: deploy-jobs-template.yml [hunk 5]
```diff
@@ -303,13 +318,43 @@ jobs:
                       },
                       {
                         "name": "FUNCTIONS_WORKER_RUNTIME",
-                        "value": "dotnet",
+                        "value": "$(FunctionsWorkerRuntime)",
```
Comment (john5macd) [inline deploy-jobs-template.yml:321]: Need to time application of FunctionsWorkerRuntime through deployment fully intend an Enabling Release which will require a deployment to test at least, if not a code change or two

**Review by john5macd — APPROVED** [review]

**Pull request merged**